### PR TITLE
Adding customizable default string to UI

### LINKED ISSUE #90: Add text field to customize the default display string
Currently, the opening display string is "Hello", which is both a little silly, and not useful for fonts not containing these characters.

It would be nice to have a text field where users could type their own.

The field would go between the drag area and the version string, and should have a label "Initial display string". The value can still be "Hello" by default, but any change the user makes should be persistent (via the app's defaults).

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #90

### CODE REVIEW

**Comment by ollimeier** [comment]
@justvanrossum Ready for review:
Screenshot 2024-03-07 at 11 45 33

**Comment by ollimeier** [comment]
@justvanrossum Read for review + a screenshot of the new design:
Screenshot 2024-03-07 at 17 39 52

**Comment by justvanrossum** [comment]
The text area is multi-style capable, I pasted some formatted text:
image

I'm afraid this may confuse users.

Can you figure out of there's a way to make it not multi-style?

**Comment by ollimeier** [comment]
> The text area is multi-style capable, I pasted some formatted text: image
> 
> I'm afraid this may confuse users.
> 
> Can you figure out of there's a way to make it not multi-style?

I'll have a look.

File: FontraPakMain.py [hunk 1]
```diff
@@ -127,6 +130,19 @@ def __init__(self, port):
...
+        self.textBox = QPlainTextEdit(self.settings.value("sampleText", "Hello"), self)
+        self.textBox.setFixedHeight(50)
+
+        self.textBox.textChanged.connect(
+            lambda: self.settings.setValue("sampleText", self.textBox.toPlainText())
+        )
+        layout.addWidget(QLabel("Initial sample text:"))
+        layout.addWidget(self.textBox)
+
+        separator = QFrame()
```
Comment (justvanrossum) [inline FontraPakMain.py:142]: To me, the window looks better without this separator.

File: FontraPakMain.py [hunk 2]
```diff
@@ -175,18 +192,21 @@ def newFont(self):
         destBackend.close()
 
         if os.path.exists(fontPath):
-            openFile(fontPath, self.port)
+            textboxValue = self.textBox.toPlainText()
+            openFile(fontPath, self.port, defaultText=textboxValue)
 
 
-def openFile(path, port):
+def openFile(path, port, defaultText="Hello"):
```
Comment (justvanrossum) [inline FontraPakMain.py:199]: Let's call this argument `sampleText`.

File: FontraPakMain.py (hunk 1 above)
Comment (ollimeier) [reply FontraPakMain.py:142]: Related to the process of design changes. With the one-liner the separator was an improvement.
Let me remove it.

File: FontraPakMain.py (hunk 2 above)
Comment (ollimeier) [reply FontraPakMain.py]: done.

**Pull request merged**